helpers/mysql/test/helpers/mysql_test.rb

```diff
@@ -93,6 +93,18 @@
       end
     end
 
+    describe 'when sql has marginalia-style prepended comments' do
+      let(:sql) do
+        '/*application:web,controller:blob,action:show,correlation_id:01EZVMR923313VV44ZJDJ7PMEZ,' \
```

Comment:
@fbogsany This may put us in a situation where developers unintentionally add PII to the marginalia comments and it is ignored by the obfuscator. 

Though it is not our total responsibility to handle ever case we should make it clear that SQL comments are going to be ignored.

Reply:
@arielvalentin - This is a great case to consider. [I've added a test to the SQL Obfuscation helper](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/529/files#diff-35551bfb3ffadde6dfff04ab16be87da9efbef7e9b82f10cd8104a141535c475R35-R47) to account for it in c28e297c 

In MySQL2 and Trilogy, the original SQL is used to get the query name (ex. 'select') and to obfuscate the statement. This leads me to believe the comments will be obfuscated even if the query name is not plucked from the beginning of the statement.

Reply:
😅 OK I am totally wrong then. glad to be.

Reply:
@kaylareopelle is correct - the obfuscation code will obfuscate comments (all comments are replaced with `?`). There is a corner case I need to address: if we choose not to obfuscate because the query is too large (perhaps because of comments 😞), we truncate the query instead, but the remaining prefix may contain PII. The "fix" may be to apply obfuscation to the truncated query, but that may run into "unmatched quotes". I can't quite recall how these things play together - I'll 👀 this more closely today.

Reply:
@fbogsany - This is an excellent callout. I prefer to address this issue in a separate PR. Would you be open to that approach?